### DISCUSSION TITLE
RA/DEC values change when the location is changed

### DISCUSSION THREAD

**Discussion by Tugikaa** [Stellarium/stellarium#3456] [Q&A] [answered]
Hi, I have a question about why the RA/DEC of an object I am observing changes when I alter the location.

This is the image showing the RA/DEC of the ISS when it passes location 1 at 18h 17m 21s UTC +7. I can confirm its coordinates at this place since I was there at that time.

<img width="1280" alt="image" src="https://github.com/Stellarium/stellarium/assets/140883009/6f6e301d-c37b-4358-ba99-b5b6055545f2">

Conversely, this is the image of the RA/DEC of the ISS when I adjust the geographic latitude and longitude to location 2. I don’t understand why its declination changes so dramatically. This picture is also timestamped at 18h 17m 21s UTC+7.

<img width="1280" alt="image" src="https://github.com/Stellarium/stellarium/assets/140883009/4eed9bdb-fd69-463d-8941-e3f67f0739fd">

Does anyone have any ideas?

Any help will be appreciated!

**Comment by gzotti** [comment, marked as answer]
ISS is 416km above surface and basically overhead at Location 1. At location 2, it is 437km away and 20° away from zenith. Also the Moon is 1° higher.  
It is called "fundamentals of 3D space".  It is the same as when you observe a car in the street from a lamp post, or from the next lamp post. At one you may see its front, possibly in front of a green house, at the other you may see its side in front of a yellow (neighboring) house.

**Reply by Tugikaa** [reply]
Thank you for your reply. I was under the impression that the observer's location only affected the altitude and azimuth. Since declination is fixed on the celestial sphere, I thought it would not change regardless of the observer's location, at a single point in time.

**Reply by 10110111** [reply]
Zenith is not a particularly useful reference point for explanation here. The main reason why RA&D change is that moving to another location moves the observer in the equatorial coordinate system with respect to a closely located object—ISS, and this changes the direction from the observer to the ISS.
If the ISS were as far as stars, its coordinates would indeed (almost) not change, since the direction would be the same (to a high precision).

**Reply by Tugikaa** [reply]
Thank you so much for your explanation!!
